### DISCUSSION TITLE
Set volume limit point

### DISCUSSION THREAD

**Discussion by MulberryPi** [rern/rAudio#1760] [Feature Requests]
Hi all,

Wondering if it's possible in the future to incorporate a set limit on the volume control? This will be very useful for those like me who use a power amp HAT, as one wrong click or touch or browser glitch on the volume control can cause the volume to go to 100 and potentially damage speakers. 

Volumio has this option in the settings and it is very reassuring to set. 

Thanks!

**Comment by MulberryPi** [comment]
Hello all, does anyone have any experience/feedback in what it takes for the devs to implement this idea? 

is there a config file modification / crude fix I can make etc to get this feature running on current rAudio systems?

This would increase usability drastically in my opinion, especially for those who use a HAT amp or the system as a preamp to drive power amps etc...

To clarify, I would like to be able to set the number value somewhere and that corresponds to the volume control not exceeding that set point. Volumio has it in their settings but I much prefer rAudio to use

**Reply by kan-ibal** [reply]
https://github.com/rern/rAudio/releases/tag/202409141

**Reply by MulberryPi** [reply]
Thank you team!
